src/client/runner/step-iterator.js

```diff
@@ -319,10 +319,13 @@ StepIterator.prototype._checkIFrame = function (element, callback) {
 
     var iFrame = domUtils.getIframeByElement(element);
 
-    this.pingIframe(iFrame, function () {
-        callback(iFrame);
-    });
-
+    this.pingIframe(iFrame)
```

Comment:
> Without catch.

I can't understand what do you mean. If `pingIframe` is rejected we should call the `callback` too